CMakeLists.txt

```diff
@@ -281,6 +301,9 @@ if ("${CMAKE_C_COMPILER_ID}" STREQUAL "GNU" OR "${CMAKE_C_COMPILER_ID}" STREQUAL
 	if ("${DISABLE_WERROR}" STREQUAL "OFF")
 	    set(CMAKE_C_FLAGS           "${CMAKE_C_FLAGS} -Werror")
 	endif()
+    if (AMIGA_M68K)
+        set(CMAKE_C_FLAGS           "${CMAKE_C_FLAGS} -fbaserel")
```

Comment:
This is absolutely required for Motorola 68k architecture.

Reply:
Ok, sure.  What is `-fbaserel` anyway?  I'm having trouble finding it in any gcc docs.

Reply:
For Motorola 68k Amiga code, it will not successfully compile without that flag.

ChatGPT can explain it better than I can.

<img width="525" alt="image" src="https://github.com/json-c/json-c/assets/6903516/60f6371a-eb74-4a15-89d7-1c2f8a6d17d3">

<img width="543" alt="image" src="https://github.com/json-c/json-c/assets/6903516/d6f1b908-55c4-4ac2-80a4-5db549416359">